src/components/next/table/expandable/arrow.css

```diff
@@ -0,0 +1,29 @@
+:host::slotted(smoothly-next-table-cell:first-child),
+:host>div:first-child {
+	position: relative;
+}
+
+:host::slotted(smoothly-next-table-cell:first-child)::before,
+:host>div:first-child::before {
```

Comment:
add the rule `transition: transform 200ms ease-in-out` use the time that feels good.